scripts/post-install.sh

```diff
@@ -51,12 +68,18 @@ https://docs.influxdata.com/influxdb/v2.0/get-started/
 EOF
 }
 
+function init_config {
+    mkdir -p $(dirname ${INFLUXD_CONFIG_PATH})
+    cat << EOF > ${INFLUXD_CONFIG_PATH}
+bolt-path = "/var/lib/influxdb/influxd.bolt"
+engine-path = "/var/lib/influxdb/engine"
+EOF
+}
+
 # Add defaults file, if it doesn't exist
 if [[ ! -s /etc/default/influxdb2 ]]; then
 cat << EOF > /etc/default/influxdb2
-INFLUXD_CONFIG_PATH=/etc/influxdb/config.toml
-INFLUXD_BOLT_PATH=/var/lib/influxdb/influxd.bolt
-INFLUXD_ENGINE_PATH=/var/lib/influxdb/engine
+INFLUXD_CONFIG_PATH=${INFLUXD_CONFIG_PATH}
```

Comment:
`influxd upgrade` will write `bolt-path` and `engine-path` into the `config.toml` output. This change + the one above are meant to make the fresh-install end state look as similar as possible to the upgrade case.